tests/suite/fixtures/fixtures.py

```diff
@@ -333,7 +335,7 @@ def cli_arguments(request) -> {}:
     print(f"Tests will run against the IC of type: {result['ic-type']}")
     if result["ic-type"] == "nginx-plus-ingress":
         print(f"Tests will use the Plus JWT")
-    result["plus-jwt"] = request.config.getoption("--plus-jwt")
+        result["plus-jwt"] = request.config.getoption("--plus-jwt")
```

Comment:
Is this necessary?

Reply:
Yes, I believe we do as we only need this if `ic-type` is `nginx-plus-ingress`